src/tools/nix-dev-shell/shell.nix

```diff
@@ -1,18 +1,22 @@
-{ pkgs ? import <nixpkgs> {} }:
-let 
-  x = import ./x { inherit pkgs; };
+{pkgs ? import <nixpkgs> {}}: let
```

Comment:
the formatting looks weird, what did you use?

Reply:
alejandra

Reply:
I think nixpkgs recently-ish switched to `nixfmt-rfc-style`, IMO it makes more sense to use it rather than